multi/imxrt-multi/imxrt-multi.c

```diff
@@ -49,6 +51,8 @@
 
 #define STACKSZ 512
 
+#define MODE_DEFAULT 0666
```

Comment:
We already have this: https://github.com/phoenix-rtos/libphoenix/blob/2a948484a6414a9363c6c33646d517f26d2b9d73/include/sys/stat.h#L76

Reply:
Thanks for it.